sys/net/gnrc/routing/rpl/gnrc_rpl.c

```diff
@@ -252,8 +252,8 @@ void _update_lifetime(void)
         parent = &gnrc_rpl_parents[i];
         if (parent->state != 0) {
             if (parent->lifetime > GNRC_RPL_LIFETIME_UPDATE_STEP) {
-                if (parent->lifetime <= (2 * GNRC_RPL_LIFETIME_UPDATE_STEP)) {
-                    gnrc_rpl_send_DIS(parent->dodag->instance, &parent->addr);
+                if (parent->lifetime > (2 * GNRC_RPL_LIFETIME_UPDATE_STEP)) {
```

Comment:
what's this change for?

Reply:
Its something I tested and forgot to remove, will do.